<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainMenu</class>
 <widget class="QWidget" name="mainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="avalibleScriptsButton">
      <property name="text">
       <string>Доступные скрипты</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewRatingsButton">
      <property name="text">
       <string>Просмотр оценок пользователей</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="settingsButton">
      <property name="text">
       <string>Настройки</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
